instagram/index.html

```diff
@@ -1,9 +1,164 @@
+                </svg>
+            </div>
+            <div class="icon">
+                <svg aria-label="저장됨" fill="#8e8e8e" height="24" viewBox="0 0 48 48" width="24">
+                    <path d="M43.5 48c-.4 0-.8-.2-1.1-.4L24 29 5.6 47.6c-.4.4-1.1.6-1.6.3-.6-.2-1-.8-1-1.4v-45C3 .7 3.7 0 4.5 0h39c.8 0 1.5.7 1.5 1.5v45c0 .6-.4 1.2-.9 1.4-.2.1-.4.1-.6.1zM24 26c.8 0 1.6.3 2.2.9l15.8 16V3H6v39.9l15.8-16c.6-.6 1.4-.9 2.2-.9z"></path>
+                </svg>
+            </div>
+            <div class="icon">
+                <svg aria-label="태그됨" fill="#8e8e8e" height="24" viewBox="0 0 48 48" width="24">
+                    <path d="M41.5 5.5H30.4c-.5 0-1-.2-1.4-.6l-4-4c-.6-.6-1.5-.6-2.1 0l-4 4c-.4.4-.9.6-1.4.6h-11c-3.3 0-6 2.7-6 6v30c0 3.3 2.7 6 6 6h35c3.3 0 6-2.7 6-6v-30c0-3.3-2.7-6-6-6zm-29.4 39c-.6 0-1.1-.6-1-1.2.7-3.2 3.5-5.6 6.8-5.6h12c3.4 0 6.2 2.4 6.8 5.6.1.6-.4 1.2-1 1.2H12.1zm32.4-3c0 1.7-1.3 3-3 3h-.6c-.5 0-.9-.4-1-.9-.6-5-4.8-8.9-9.9-8.9H18c-5.1 0-9.4 3.9-9.9 8.9-.1.5-.5.9-1 .9h-.6c-1.7 0-3-1.3-3-3v-30c0-1.7 1.3-3 3-3h11.1c1.3 0 2.6-.5 3.5-1.5L24 4.1 26.9 7c.9.9 2.2 1.5 3.5 1.5h11.1c1.7 0 3 1.3 3 3v30zM24 12.5c-5.3 0-9.6 4.3-9.6 9.6s4.3 9.6 9.6 9.6 9.6-4.3 9.6-9.6-4.3-9.6-9.6-9.6zm0 16.1c-3.6 0-6.6-2.9-6.6-6.6 0-3.6 2.9-6.6 6.6-6.6s6.6 2.9 6.6 6.6c0 3.6-3 6.6-6.6 6.6z"></path>
+                </svg>
+            </div>
+        </div>
+        <div class="feed-container">
+            <div class="feed-line">
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+            </div>
+            <div class="feed-line">
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+            </div>
+            <div class="feed-line">
+                <article class="feed">
+                    <img src="img/feed.jpeg">
+                </article>
+            </div>
+        </div>
+    </div>
```

Comment:
`flex-wrap` 속성을 이용해서 다음과 같이 바꿀 수 있으면 좋겠네요!

```suggestion
          <div class="feed-container">
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
            <article class="feed">
                <img src="img/feed.jpeg">
            </article>
        </div>
```